### PR TITLE
Predictions page

### PR DESCRIPTION
**Pull request by BenNewman90**
I've added a couple more queries to the database and a table into the predictions page.

### CODE REVIEW

File: client/src/components/App.js [hunk 1]
```diff
@@ -22,20 +22,20 @@ class App extends Component {
...
-        console.log(data, "DATA");
         this.setState({ users: data[0], matches: data[1], teams: data[2] });
+        console.log(this.state.matches);
       })
       .catch(err => console.log(err));
   }
 
   render() {
-    if (this.state.users.length === 0) {
+    if (this.state.teams.length !== 32) {
```
Comment (Karyum) [inline client/src/components/App.js]: will this always be 32?

File: client/src/components/App.js [hunk 2]
```diff
@@ -22,20 +22,20 @@ class App extends Component {
     ])
       .then(res => Promise.all(res.map(dataset => dataset.json())))
       .then(data => {
-        console.log(data, "DATA");
         this.setState({ users: data[0], matches: data[1], teams: data[2] });
+        console.log(this.state.matches);
```
Comment (Karyum) [inline client/src/components/App.js]: there shouldnt be any console logs in the frontend

File: client/src/components/Predictions.js [hunk 3]
```diff
@@ -1,23 +1,67 @@
 import React, { Component } from "react";
 import styled from "styled-components";
-import Row from "./Row";
+// import bennyboy from "../assets/profilePhotos/bennyboy.jpeg";
```
Comment (Karyum) [inline client/src/components/Predictions.js]: why is this commented out ?

File: client/src/components/Predictions.js [hunk 4]
```diff
@@ -1,23 +1,67 @@
...
 const StyledPredictionsDiv = styled.div`
   position: fixed;
   top: 75px;
 `;
 class Predictions extends Component {
+  createTable = () => {
+    let table = [];
+    const { users, matches } = this.props;
+    // Outer loop to create parent
+    for (let i = 0; i < users.length + 1; i++) {
```
Comment (Karyum) [inline client/src/components/Predictions.js]: everytime that you are looping over something's length, then it is better to use `.forEach` cause its cleaner and easier to read

File: client/src/components/Predictions.js [hunk 5]
```diff
@@ -1,23 +1,67 @@
...
+      let children = [];
+      //Inner loop to create children
+      for (let j = 0; j < matches.length + 3; j++) {
```
Comment (Karyum) [inline client/src/components/Predictions.js]: :arrow_up:

File: client/src/index.css [hunk 6]
```diff
@@ -1,5 +1,19 @@
...
+
+table {
+  background-color:#EDE9E9;
+}
+
+.tbl-points {
+    background-color:#C3C2C2;
+}
+
+.tbl-header{
```
Comment (Karyum) [inline client/src/index.css]: hat happened to styled components ?

File: client/src/components/App.js (hunk 1 above)
Comment (BenNewman90) [reply client/src/components/App.js]: For the world cup yes, but for other tournaments no. I will change it.

File: client/src/components/App.js (hunk 2 above)
Comment (BenNewman90) [reply client/src/components/App.js]: done.

File: client/src/components/Predictions.js (hunk 5 above)
Comment (BenNewman90) [reply client/src/components/Predictions.js]: Didn't change it, as discussed

File: client/src/components/Predictions.js (hunk 4 above)
Comment (BenNewman90) [reply client/src/components/Predictions.js]: no changes, as discussed

File: client/src/components/Predictions.js (hunk 3 above)
Comment (BenNewman90) [reply client/src/components/Predictions.js]: tried to store the photos locally. didnt work, but will try again.

**Pull request merged**